models/Link.yaml

```diff
@@ -57,10 +57,6 @@ fields:
     options:
```

Comment:
if link cannot be changed, maybe it is better to remove this field altogether?

Reply:
@ScottyMJacobson said he'll take care of cleaning it up :)

Reply:
cool

Reply:
@smnh how concerned are we about backwards-compatibility right now? will projects that have a `Link` with this `style` field break or just print an error?

Reply:
@ScottyMJacobson I tested this in Studio and style field was removed.

It seems like to release this safely the following things still need to be done.

* Can you please check the component itself and add the proper guards to link and button if the style prop is missing?
* Can you also update any content in `stackbit-content-packs` and remove the style field.